tests/insights-api-tests/functionsAnalytics.spec.ts

```
@@ -5,9 +5,9 @@ import {
   sendGraphqlQuery,
   getReferenceDataForCountry,
   getConsumersAndPushToAnnotations,
-} from "../helper";
+} from "../helpers/main-helper";
 import { fileURLToPath } from "url";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)